cloud_init.init

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+sudo yum install -y https://s3.amazonaws.com/ec2-downloads-windows/SSMAgent/latest/linux_amd64/amazon-ssm-agent.rpm
```

Comment:
How about pinning the version here? Instead of `latest`

Reply:
Latest ist totally fine for this use-case.  AWS almost always keeps backwards compatibility.